### PR TITLE
Suppress ffmpeg output by default and add verbose argument to display it

### PR DESCRIPTION
**Pull request by alik-git**
This can be tested with the example script provided and any krec.mkv file

### CODE REVIEW

**Review by hatomist — APPROVED** [review]

File: examples/test_krec_read.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+def main(args: argparse.Namespace) -> None:
+
+    print(f"Reading KRec file: {args.krec_file}")
+    krec_obj = load_krec(args.krec_file)
+    print(f"krec_obj: {krec_obj}")
+    print(f"krec_obj.header: {krec_obj.header}")
+    print(f"num frames: {len(krec_obj)}")
+    print(f"succesfully loaded KRec file!")
```
Comment (codekansas) [inline examples/test_krec_read.py:67]: don't use print statements

use logging

Comment (hatomist) [reply examples/test_krec_read.py:67]: to be fair, not a library code but just an example script

Comment (alik-git) [reply examples/test_krec_read.py:67]: yes, I did use logging in the library code, for example see here

https://github.com/kscalelabs/krec/blob/cdaae80a8677a7b950092bf7f6e7ac2955732c72/src/ffmpeg.rs#L89

but if we don't want print statements in the example files, I'm happy to change that as well.

Comment (codekansas) [reply examples/test_krec_read.py:67]: we should use python logging everywhere

**Pull request merged**